lib/projectLink.js

```diff
@@ -10,13 +10,24 @@ class ProjectLink {
   }
 
   async delete(db) {
-    await db.deleteLink(this.project, this.label, this.url);
+    await db.deleteLink(this.project, this.label);
+    console.log('deleted');
```

Comment:
I have to delete this console.log... i didnt catch it!

Reply:
Don't forge to remove this.